lib/ricohapi/mstorage/client.rb

```diff
@@ -95,7 +95,18 @@ def add_meta(media_id, user_meta)
 
       # DELETE /media/{id}/meta/user, DELETE /media/{id}/meta/user/{key}
       def remove_meta(media_id, key)
-        # TODO: do something
+        case key
+        when 'user'
+          handle_response(:as_raw) do
+            token.delete endpoint_for("media/#{media_id}/meta/user")
+          end
+        when USER_META_REGEX
+          handle_response(:as_raw) do
+            token.delete endpoint_for("media/#{media_id}/meta/user/#{$1}")
+          end
+        else
+          raise Error.new("invalid key: #{key.inspect}")
```

Comment:
I found the same code at here. > https://github.com/hishida/media-storage-rb/blob/cd8f4767ad4a114e940b57f1e56204249f49a113/lib/ricohapi/mstorage/client.rb#L73
What are we gonna do?